src/js/variant-widget/filters/eva-form-panel.js

```diff
@@ -193,5 +193,31 @@ EvaFormPanel.prototype = {
         });
         return form;
 
+    },
+    validatePositionFilter : function(values) {
+        var validate = true;
+        _.each(values, function (region) {
+            if(region.match(/^[\w\d]+\:\d+\-\d+/)){
```

Comment:
According to the VCF spec: 

> Valid contig names must follow the reference sequence names allowed by the SAM format ("[!-)+-<>-~][!-~]\*") excluding the characters "<>[]:*"

So you might be able to use that regular expression with the exclusion. Please implement some tests to check values are accepted or rejected accordingly.